server/routes/wishCards.js

```diff
@@ -369,11 +369,11 @@ router.get('/:id', redirectLogin, getByIdValidationRules(), validate, async (req
 router.get('/donate/:id', redirectLogin, getByIdValidationRules(), redirectLogin, async (req, res) => {
   try {
     const wishcard = await WishCardRepository.getWishCardByObjectId(req.params.id);
-   
-    // NOTE: 
+
+    // NOTE:
     // this agency object is returning undefined and breaking frontend
-    const agency = wishcard.wishCardTo;
-    
+    const agency = wishcard.belongsTo;
+
     console.log("test print");
```

Comment:
I think someone added this for testing (can be removed now with seeder + migrations)

Reply:
done